.golangci.yml

```diff
@@ -60,6 +60,18 @@ linters-settings:
       # specify an error message to output when a blacklisted package is used
       - log: "logging is allowed only by pingcap/log"
       - github.com/juju/errors: "error handling is allowed only by pingcap/errors"
+    additional-guards:
+      - list-type: denylist
+        include-go-root: true
+        packages:
+          - errors
+          - github.com/pingcap/errors
+        packages-with-error-message:
+          - errors: "error handling is allowed only by github.com/pingcap/tiflow/pkg/errors"
+          - github.com/pingcap/errors: "error handling is allowed only by github.com/pingcap/tiflow/pkg/errors"
+        ignore-file-rules:
+          - "!**/tiflow/engine/**"
+          - "**/engine/pkg/rpcutil/*.go"
```

Comment:
This rule is confused, I wonder why `!**/tiflow/cdc/**` is not configured, and it is not checked.

Reply:
1. `"!**/tiflow/engine/**"` means all files except `engine` directory should be excluded.
2. `"**/engine/pkg/rpcutil/*.go"` means `engine/pkg/rpcutil` directory should be excluded.

`tiflow/cdc` is excluded in rule 1.